src/extras/core/Path.js

```diff
@@ -151,7 +151,8 @@ THREE.Path.prototype = Object.assign( Object.create( THREE.CurvePath.prototype )
 			xRadius, yRadius,
 			aStartAngle, aEndAngle,
 			aClockwise,
-			aRotation || 0 // aRotation is optional.
+			aRotation || 0, // aRotation is optional.
+			this.curves.length
```

Comment:
In the original code, the calculation of the point on the ellipse was made twice. In EllipseCurve.getPoint, as I do, and in this code. I don’t want to have the same thing calculated in two different codes, so I need a way to call EllipseCurve.getPoint for the case in the getPoints function a few lines below.
As the index in actions and curves are different for an element of the path, I needed to store the index of the ellipse in curves somewhere. actions is exactly done for that. So I added the index of the ellipse in args, and as the last two elements of args must be the coordinates of the last point of the ellipse, I added just after aRotation. At that moment the ellipse is not added to curves, so the (future) index of the ellipse in the curves array is curves.length.